### ISSUE TITLE
Extra 0 visible in front (login) page

### ISSUE THREAD

**Issue by tahvane1** [keycloakify/keycloakify#627]
I copied  starter project. Changed theme name and build. End result contains extra 0 in front page.

```html
<div id="kc-content">
    <div id="kc-content-wrapper">
        <div id="kc-form">
            <div id="kc-form-wrapper">
                <form id="kc-form-login" action="<removed>" method="post">
                    <div class="kcFormGroupClass form-group"><label for="username"
                            class="kcLabelClass pf-c-form__label pf-c-form__label-text">
                            <div data-kc-msg="usernameOrEmail">Username or email</div>
                        </label><input tabindex="2" id="username" class="kcInputClass pf-c-form-control" name="username"
                            type="text" autocomplete="off" aria-invalid="false" value=""></div>
                    <div class="kcFormGroupClass form-group kcFormSettingClass login-pf-settings">
                        <div id="kc-form-options">
                            <div class="checkbox"><label><input tabindex="3" id="rememberMe" name="rememberMe"
                                        type="checkbox">
                                    <div data-kc-msg="rememberMe">Remember me</div>
                                </label></div>
                        </div>
                    </div>
                    <div id="kc-form-buttons" class="kcFormGroupClass form-group"><input tabindex="4"
                            class="kcButtonClass pf-c-button kcButtonPrimaryClass pf-m-primary kcButtonBlockClass pf-m-block kcButtonLargeClass btn-lg"
                            name="login" id="kc-login" type="submit" value="Sign In"></div>
                </form>
            </div>
        </div>0
    </div>
</div>
```

This part `</div>0</div>`

**Comment by garronej** [comment]
Hello @tahvane1,  

It's strange, I can't reproduce.  

Are you sure you didn't eject the login page and added a 0 here by mistake?  

<img width="1177" alt="image" src="https://github.com/user-attachments/assets/fcfcd61a-75d5-4cc8-ab5d-0507ab193d4c">

Looking at the code I don't understand how it's possible otherwise...  

If not what is the name that you gave to the theme? 
If you remove themeName from your configuration do you still get the error?

**Comment by tahvane1** [comment]
I'm sure that I did not inject login page (as I started from scratch when I noticed problem). I'm using latest keycloak and in first phase only login name is asked... I will check if it happens with default name.

**Comment by tahvane1** [comment]
I cloned starter project. Build and deployed, changed theme. Same thing.

**Comment by garronej** [comment]
I think there might be something hapening with the locale that are defined on your realm.  

Can you show me what there is in your: `window.kcContext["x-keycloakify"].messages`

Like:  

![image](https://github.com/user-attachments/assets/79ca6a1b-1ee3-4d7b-b78b-41f2afadb18c)

**Comment by garronej** [comment]
Can you try with the official jar of the starter:  

https://github.com/keycloakify/keycloakify-starter/releases/tag/v10.0.4  

(The keycloak-theme-for-kc-22-and-above.jar)  

If you don't see the issue can you try:  

```tsx
git clone https://github.com/keycloakify/keycloakify-starter
cd keycloakify-starter
yarn
npx keycloakify start-keycloak
```

And tell me if you see the issue.

**Comment by tahvane1** [comment]
by using  jar in kubernetes yes.  Using docker no.

**Comment by garronej** [comment]
Ok so the bug must be triggered by some configuration on your keycloak server.  

Can you please compy the full `window.kcContext` that you're getting on the login page?  

Also a screenshot of the page that would help me figure out what's going on.  

Thanks

**Comment by tahvane1** [comment]
`{
    "social": {
        "displayInfo": false,
        "providers": []
    },
    "auth": {
        "showResetCredentials": false,
        "authenticationSelections": [
            {
                "helpText": "organization-help-text",
                "displayName": "organization-display-name",
                "iconCssClass": "kcAuthenticatorDefaultClass",
                "authExecId": "407723d0-4b60-4730-84e7-6b25afd44dae",
                "authenticationExecution": {
                    "authenticatorFlow": false,
                    "conditional": false,
                    "required": false,
                    "enabled": true,
                    "disabled": false,
                    "id": "407723d0-4b60-4730-84e7-6b25afd44dae",
                    "parentFlow": "fc831d89-d91c-41c5-866c-55ad21cc6437",
                    "authenticator": "organization",
                    "alternative": true,
                    "requirement": {
                        "stableIndex": 2
                    },
                    "priority": 1
                }
            },
            {
                "helpText": "auth-username-password-form-help-text",
                "displayName": "auth-username-password-form-display-name",
                "iconCssClass": "kcAuthenticatorDefaultClass",
                "authExecId": "ed879563-e2f7-4bd9-9ca2-92a04160e2ae",
                "authenticationExecution": {
                    "authenticatorFlow": false,
                    "conditional": false,
                    "required": true,
                    "enabled": true,
                    "disabled": false,
                    "id": "ed879563-e2f7-4bd9-9ca2-92a04160e2ae",
                    "parentFlow": "964c7020-afe2-4952-933d-1f887998a8f4",
                    "authenticator": "auth-username-password-form",
                    "alternative": false,
                    "requirement": {
                        "stableIndex": 0
                    },
                    "priority": 10
                }
            },
            {
                "helpText": "webauthn-passwordless-help-text",
                "displayName": "webauthn-passwordless-display-name",
                "iconCssClass": "kcAuthenticatorWebAuthnPasswordlessClass",
                "authExecId": "160b9597-9f58-4ff2-81e9-35fae444e75b",
                "authenticationExecution": {
                    "authenticatorFlow": false,
                    "conditional": false,
                    "required": false,
                    "enabled": true,
                    "disabled": false,
                    "id": "160b9597-9f58-4ff2-81e9-35fae444e75b",
                    "parentFlow": "bbf79e08-1ae0-4de3-9fd5-b498e6eedb6b",
                    "authenticator": "passkeys-authenticator",
                    "alternative": true,
                    "requirement": {
                        "stableIndex": 2
                    },
                    "priority": 32
                }
            }
        ],
        "showUsername": false,
        "showTryAnotherWayLink": false
    },
    "locale": {
        "current": "English",
        "supported": [
            {
                "languageTag": "en",
                "label": "English",
                "url": "/realms/meo/login-actions/authenticate?client_id=SN4MEO&tab_id=FxeEXtNiPD0&client_data=eyJydSI6Imh0dHBzOi8vbWVvdGVzdC5jb25maWRlbnRseS5maS9TTjRNRU8vb2lkY19jYWxsYmFja19oYW5kbGVyIiwicnQiOiJjb2RlIn0&execution=407723d0-4b60-4730-84e7-6b25afd44dae&kc_locale=en"
            },
            {
                "languageTag": "fi",
                "label": "Suomi",
                "url": "/realms/meo/login-actions/authenticate?client_id=SN4MEO&tab_id=FxeEXtNiPD0&client_data=eyJydSI6Imh0dHBzOi8vbWVvdGVzdC5jb25maWRlbnRseS5maS9TTjRNRU8vb2lkY19jYWxsYmFja19oYW5kbGVyIiwicnQiOiJjb2RlIn0&execution=407723d0-4b60-4730-84e7-6b25afd44dae&kc_locale=fi"
            },
            {
                "languageTag": "sv",
                "label": "Svenska",
                "url": "/realms/meo/login-actions/authenticate?client_id=SN4MEO&tab_id=FxeEXtNiPD0&client_data=eyJydSI6Imh0dHBzOi8vbWVvdGVzdC5jb25maWRlbnRseS5maS9TTjRNRU8vb2lkY19jYWxsYmFja19oYW5kbGVyIiwicnQiOiJjb2RlIn0&execution=407723d0-4b60-4730-84e7-6b25afd44dae&kc_locale=sv"
            }
        ],
        "currentLanguageTag": "en"
    },
    "login": {},
    "url": {
        "oauthAction": "/realms/meo/login-actions/authenticate",
        "resourcesPath": "/resources/0pzc5/login/keycloakify-starter",
        "loginAction": "https://auth.confidently.fi/realms/meo/login-actions/authenticate?session_code=18BeRuWwajiW1sz6CsEIUDT55x5YmpG3AQAuZUbYSjI&execution=407723d0-4b60-4730-84e7-6b25afd44dae&client_id=SN4MEO&tab_id=FxeEXtNiPD0&client_data=eyJydSI6Imh0dHBzOi8vbWVvdGVzdC5jb25maWRlbnRseS5maS9TTjRNRU8vb2lkY19jYWxsYmFja19oYW5kbGVyIiwicnQiOiJjb2RlIn0",
        "loginUrl": "/realms/meo/login-actions/authenticate?client_id=SN4MEO&tab_id=FxeEXtNiPD0&client_data=eyJydSI6Imh0dHBzOi8vbWVvdGVzdC5jb25maWRlbnRseS5maS9TTjRNRU8vb2lkY19jYWxsYmFja19oYW5kbGVyIiwicnQiOiJjb2RlIn0",
        "resourcesCommonPath": "/resources/0pzc5/login/keycloakify-starter/resources-common",
        "firstBrokerLoginUrl": "/realms/meo/login-actions/first-broker-login?client_id=SN4MEO&tab_id=FxeEXtNiPD0&client_data=eyJydSI6Imh0dHBzOi8vbWVvdGVzdC5jb25maWRlbnRseS5maS9TTjRNRU8vb2lkY19jYWxsYmFja19oYW5kbGVyIiwicnQiOiJjb2RlIn0",
        "registrationUrl": "/realms/meo/login-actions/registration?client_id=SN4MEO&tab_id=FxeEXtNiPD0&client_data=eyJydSI6Imh0dHBzOi8vbWVvdGVzdC5jb25maWRlbnRseS5maS9TTjRNRU8vb2lkY19jYWxsYmFja19oYW5kbGVyIiwicnQiOiJjb2RlIn0",
        "logoutConfirmAction": "/realms/meo/protocol/openid-connect/logout/logout-confirm?client_id=SN4MEO&tab_id=FxeEXtNiPD0&client_data=eyJydSI6Imh0dHBzOi8vbWVvdGVzdC5jb25maWRlbnRseS5maS9TTjRNRU8vb2lkY19jYWxsYmFja19oYW5kbGVyIiwicnQiOiJjb2RlIn0",
        "registrationAction": "https://auth.confidently.fi/realms/meo/login-actions/authenticate?session_code=18BeRuWwajiW1sz6CsEIUDT55x5YmpG3AQAuZUbYSjI&execution=407723d0-4b60-4730-84e7-6b25afd44dae&client_id=SN4MEO&tab_id=FxeEXtNiPD0&client_data=eyJydSI6Imh0dHBzOi8vbWVvdGVzdC5jb25maWRlbnRseS5maS9TTjRNRU8vb2lkY19jYWxsYmFja19oYW5kbGVyIiwicnQiOiJjb2RlIn0",
        "loginResetCredentialsUrl": "/realms/meo/login-actions/reset-credentials?client_id=SN4MEO&tab_id=FxeEXtNiPD0&client_data=eyJydSI6Imh0dHBzOi8vbWVvdGVzdC5jb25maWRlbnRseS5maS9TTjRNRU8vb2lkY19jYWxsYmFja19oYW5kbGVyIiwicnQiOiJjb2RlIn0",
        "ssoLoginInOtherTabsUrl": "/realms/meo/login-actions/restart?client_id=SN4MEO&tab_id=FxeEXtNiPD0&client_data=eyJydSI6Imh0dHBzOi8vbWVvdGVzdC5jb25maWRlbnRseS5maS9TTjRNRU8vb2lkY19jYWxsYmFja19oYW5kbGVyIiwicnQiOiJjb2RlIn0&skip_logout=true",
        "resourcesUrl": "/resources/0pzc5?client_id=SN4MEO&tab_id=FxeEXtNiPD0&client_data=eyJydSI6Imh0dHBzOi8vbWVvdGVzdC5jb25maWRlbnRseS5maS9TTjRNRU8vb2lkY19jYWxsYmFja19oYW5kbGVyIiwicnQiOiJjb2RlIn0/login/keycloakify-starter",
        "loginRestartFlowUrl": "/realms/meo/login-actions/restart?client_id=SN4MEO&tab_id=FxeEXtNiPD0&client_data=eyJydSI6Imh0dHBzOi8vbWVvdGVzdC5jb25maWRlbnRseS5maS9TTjRNRU8vb2lkY19jYWxsYmFja19oYW5kbGVyIiwicnQiOiJjb2RlIn0&skip_logout=false",
        "oauth2DeviceVerificationAction": "/realms/meo/login-actions/authenticate"
    },
    "authenticationSession": {
        "tabId": "FxeEXtNiPD0",
        "authSessionId": "5085c592-6411-4468-89d3-4c9d1e007aaf"
    },
    "messagesPerField": {},
    "client": {
        "clientId": "SN4MEO",
        "description": "",
        "baseUrl": "",
        "name": "SN4MEO",
        "attributes": {
            "oidc.ciba.grant.enabled": "false",
            "client.secret.creation.time": "1724230027",
            "backchannel.logout.session.required": "true",
            "post.logout.redirect.uris": "http://meotest.confidently.fi/SN4MEO/index.faces",
            "oauth2.device.authorization.grant.enabled": "false",
            "display.on.consent.screen": "false",
            "backchannel.logout.revoke.offline.tokens": "false"
        }
    },
    "passwordPolicies": {
        "notEmail": false,
        "notUsername": false
    },
    "realm": {
        "displayName": "meo",
        "registrationAllowed": false,
        "displayNameHtml": "<div class=\"confidently-logo-text\"/>",
        "resetPasswordAllowed": true,
        "resetCredentialsActionTokenLifespanMinutes": 5,
        "password": true,
        "actionTokenGeneratedByUserLifespanMinutes": 5,
        "internationalizationEnabled": true,
        "idpVerifyAccountLinkActionTokenLifespanMinutes": 5,
        "verifyEmailActionTokenLifespanMinutes": 5,
        "loginWithEmailAllowed": true,
        "duplicateEmailsAllowed": false,
        "editUsernameAllowed": false,
        "name": "meo",
        "rememberMe": true,
        "registrationEmailAsUsername": false,
        "identityFederationEnabled": true
    },
    "scripts": [],
    "properties": {},
    "keycloakifyVersion": "10.0.4",
    "themeVersion": "10.0.4",
    "themeType": "login",
    "themeName": "keycloakify-starter",
    "pageId": "login-username.ftl",
    "ftlTemplateFileName": "login-username.ftl",
    "x-keycloakify": {
        "resourcesPath": "/resources/0pzc5/login/keycloakify-starter",
        "messages": {
            "auth-username-password-form-help-text": "Sign in by entering your username and password.",
            "auth-username-password-form-display-name": "Username and password",
            "webauthn-passwordless-help-text": "Use your Passkey for passwordless sign in.",
            "webauthn-passwordless-display-name": "Passkey"
        }
    }
}`

![image](https://github.com/user-attachments/assets/1f2d027a-a8ec-422a-9495-337f560ad4ad)

**Comment by garronej** [comment]
Ok, I understand what's going on, I'll fix it

**Comment by garronej** [comment]
Ok I fixed it in 10.0.5  

Thank you for helping me figuring it out.

**Comment by tahvane1** [comment]
Thanks. It is working now.
